[2024-10-02T18:59:43.969+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T18:59:43.991+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-30T00:00:00+00:00 [queued]>
[2024-10-02T18:59:43.999+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-30T00:00:00+00:00 [queued]>
[2024-10-02T18:59:43.999+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T18:59:44.186+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-01-30 00:00:00+00:00
[2024-10-02T18:59:44.191+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6396) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T18:59:44.192+0000] {standard_task_runner.py:63} INFO - Started process 6400 to run task
[2024-10-02T18:59:44.192+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-01-30T00:00:00+00:00', '--job-id', '1339', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpcvhg4lmh']
[2024-10-02T18:59:44.194+0000] {standard_task_runner.py:91} INFO - Job 1339: Subtask carregar
[2024-10-02T18:59:44.232+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-30T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T18:59:44.307+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-30T00:00:00+00:00'
[2024-10-02T18:59:44.308+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T18:59:44.362+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T18:59:44.363+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T18:59:44.372+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-01-30T00:00:00+00:00, execution_date=20240130T000000, start_date=20241002T185943, end_date=20241002T185944
[2024-10-02T18:59:44.406+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T18:59:44.420+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T18:59:44.423+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-10-02T19:25:50.626+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:25:50.954+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-30T00:00:00+00:00 [queued]>
[2024-10-02T19:25:50.968+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-30T00:00:00+00:00 [queued]>
[2024-10-02T19:25:50.969+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:25:50.993+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carregar> on 2024-01-30 00:00:00+00:00
[2024-10-02T19:25:51.004+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9726) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-02T19:25:51.006+0000] {standard_task_runner.py:63} INFO - Started process 9745 to run task
[2024-10-02T19:25:51.007+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_covid_europa_pipeline', 'carregar', 'scheduled__2024-01-30T00:00:00+00:00', '--job-id', '2336', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpy91grr43']
[2024-10-02T19:25:51.009+0000] {standard_task_runner.py:91} INFO - Job 2336: Subtask carregar
[2024-10-02T19:25:51.076+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_covid_europa_pipeline.carregar scheduled__2024-01-30T00:00:00+00:00 [running]> on host 99de7dfc3871
[2024-10-02T19:25:51.229+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_covid_europa_pipeline' AIRFLOW_CTX_TASK_ID='carregar' AIRFLOW_CTX_EXECUTION_DATE='2024-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-30T00:00:00+00:00'
[2024-10-02T19:25:51.231+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:25:51.358+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:25:51.359+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:25:51.373+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_covid_europa_pipeline, task_id=carregar, run_id=scheduled__2024-01-30T00:00:00+00:00, execution_date=20240130T000000, start_date=20241002T192550, end_date=20241002T192551
[2024-10-02T19:25:51.422+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-02T19:25:51.446+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:25:51.456+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
